packages/pn-commons/src/components/FileUpload.tsx

```diff
@@ -309,10 +342,12 @@ const FileUpload = ({
             alignItems="center"
             sx={{ width: '100%' }}
           >
-            <Box display="flex" justifyContent="center" alignItems="center">
-              <AttachFileIcon color="primary" />
-              <Typography color="primary">{fileData.file.name}</Typography>
-              <Typography fontWeight={600} sx={{ marginLeft: '30px' }}>
+            <Box display={isMobile ? 'block' : 'flex'} justifyContent="center" alignItems="center">
```

Comment:
IMHO the `justifyContent` is of no use for this Box. In fact, the clip + file name are expected to be left-aligned. We could take the opportunity to erase it.